server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -89,7 +90,7 @@ const orderMutations = {
         paymentMethod,
         fromCollective: fromCollective && { id: fromCollective.id },
         collective: { id: collective.id },
-        totalAmount: getOrderTotalAmount(order),
+        totalAmount: paymentMethod.type === PAYMENT_METHOD_TYPE.CRYPTO ? 1 : getOrderTotalAmount(order), // insert 1 as a dummy value for the pending order for crypto
```

Comment:
That should not be needed. Frontend can simply pass a random value. No need to pollute this.

Reply:
Makes sense, I've corrected this. Actually I don't think this is needed anymore. 👍🏽